conda/chrisalis.yml

```diff
@@ -0,0 +1,34 @@
+name: zppy_dev
```

Comment:
The contents of this file should be included into https://github.com/E3SM-Project/zppy-interfaces/blob/main/conda/dev.yml